src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -549,19 +549,25 @@ private void ToggleGesture(RoutedEvent routedEvent)
 		partial void PrepareManagedGestureEventBubbling(RoutedEvent routedEvent, ref RoutedEventArgs args, ref BubblingMode bubblingMode)
 		{
 			// When we bubble a gesture event from a child, we make sure to abort any pending gesture/manipulation on the current element
-			if (routedEvent == HoldingEvent)
+			if (IsGestureRecognizerCreated)
 			{
-				if (IsGestureRecognizerCreated)
+				if (routedEvent == TappedEvent)
+				{
+					GestureRecognizer.PreventEvents(((TappedRoutedEventArgs)args).PointerId, GestureSettings.Tap);
+				}
+				else if (routedEvent == DoubleTappedEvent)
+				{
+					GestureRecognizer.PreventEvents(((DoubleTappedRoutedEventArgs)args).PointerId, GestureSettings.DoubleTap);
+				}
+				else if (routedEvent == RightTappedEvent)
+				{
+					GestureRecognizer.PreventEvents(((RightTappedRoutedEventArgs)args).PointerId, GestureSettings.RightTap);
+				}
+				else if (routedEvent == HoldingEvent)
```

Comment:
We should add a feature flag for this, it might cause some regression and it's easy to add one here.

BTW: Great that you are able to disable only using the pointer ID 👍

Reply:
Sure we will have to and the flag would be configured to enable the new behavior by default to detect those. But providing a feature flag allows users to still upgrade uno without being blocked by a regression caused by this and reduces the pressure on our team to fix them.

Reply:
Ah, so it's a "just in case" there is a regression? Sounds good to me. Should we plan removing that flag in Uno 6 if no regressions were revealed?

Reply:
Well currently we don't have any well formalized way to remove those flags ... I would say that the easiest would be to create an new issue :)